finder-app/manual-linux.sh

```diff
@@ -13,6 +13,8 @@ FINDER_APP_DIR=$(realpath $(dirname $0))
 ARCH=arm64
 CROSS_COMPILE=aarch64-none-linux-gnu-
 
+WRITER_PATH=/home/dhruv/Documents/AESD/GitHub/Assignment_1/assignment-1-DhruvHMehta/finder-app
```

Comment:
@DhruvHMehta this can probably be deleted, won't be relevant for qemu images.

Reply:
Yes, sorry I forgot to remove this after I used FINDER_APP_DIR for the path.